perfeed/llms/ollama_client.py

```diff
@@ -37,7 +35,7 @@ def chat_completion(self, system: str, user: str, **kwargs) -> str:
                 {"role": "user", "content": user},
             ],
             options={
-                "num_ctx": kwargs.get("num_ctx", 4096 * 2),
+                "num_ctx": kwargs.get("num_ctx", 1024 * 32),
```

Comment:
we can just set it to 128000 and test models had smaller window to see if u get parameter error. I got a feeling it will allow it anyways then u get to set the maximum of the windows

Reply:
Why 128K? Does it max out for most commercial laptops?

Reply:
128k is like a standard size or at least a size ppl expect the window to be for llms. anything smaller than, you can find a finetuned version of 128k.